python/graphstorm/config/argument.py

```diff
@@ -480,15 +480,19 @@ def _check_fanout(self, fanout, fot_name):
             else:
                 # Fanout in format of
                 # etype2:20@etype3:20@etype1:20,etype2:10@etype3:4@etype1:2
+                # Each etype should be a canonical etype in format of
+                # srcntype_relation_dstntype
 
-                fanout = [{k.split(":")[0]: int(k.split(":")[1]) \
+                fanout = [{tuple(k.split(":")[0].split('_')): int(k.split(":")[1]) \
```

Comment:
what is this underscore?

Reply:
Any other suggestions? The `,` and `:` are used. how about `;` ?

Reply:
let's not use underscore. we can potentially use `|`, '/'. '_' is commonly used in names.